README.md

```diff
@@ -45,9 +92,11 @@ Give this up to 1 minute to start up. You can check the Airflow web-server healt
 $ docker-compose -p infra ps
 ```
 
-#### Reload Docker Instances
+**NOTE** You may encounter a `Variale Import Failed` error which will prevent the proper startup of all the Docker containers.  This is a common problem.  See [Variable Import Failed](#VariableImportFailed) below and re-run `make-up`
```

Comment:
I can leave this in here, but I have not run into this problem when working in this docker environment.
```suggestion
**NOTE** You may encounter a `Variable Import Failed` error which will prevent the proper startup of all the Docker containers.  This is a common problem.  See [Variable Import Failed](#VariableImportFailed) below and re-run `make-up`
```

Reply:
Rewrtten